src/main/java/org/gridsuite/filter/identifierlistfilter/IdentifierListFilter.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.filter.identifierlistfilter;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+import org.gridsuite.filter.AbstractFilter;
+import org.gridsuite.filter.utils.EquipmentType;
+import org.gridsuite.filter.utils.FilterType;
+
+import java.util.*;
+
+
+/**
+ * @author Seddik Yengui <seddik.yengui at rte-france.com>
+ */
+
+@Getter
+@Schema(description = "Identifier list Filters", allOf = AbstractFilter.class)
+@SuperBuilder
+public class IdentifierListFilter extends AbstractFilter {
+
+    private List<IdentifierListFilterEquipmentAttributes> filterEquipmentsAttributes;
+
+    public IdentifierListFilter() {
+    }
+
+    public IdentifierListFilter(UUID id,
+                                Date modificationDate,
+                                EquipmentType equipmentType,
+                                List<IdentifierListFilterEquipmentAttributes> filterEquipmentsAttributes) {
+        super(id, modificationDate, equipmentType);
+        this.filterEquipmentsAttributes = filterEquipmentsAttributes;
+    }
```

Comment:
Replace with `@AllArgConstructor`

Reply:
No

Reply:
I forgot that lombok don't support `super()`.